src/structure/spatial.jl

```diff
@@ -187,6 +187,23 @@ function omegaangle(res::AbstractResidue, res_prev::AbstractResidue)
 end
 
 """
+Calculate the omega angle in radians for a residue in a `Chain`. Arguments are
+the chain and the first residue index (where atoms "N" and "CA" are taken
+from). The next residue is at pos + 1 (where atom "CA" and "C" iares required).
+"""
+function omegaangle(chain::Chain, pos::Int64)
+    if !(string(pos) in keys(chain.residues))
+        throw(ArgumentError("$pos is an invalid index"))
+    elseif !(string(pos + 1) in keys(chain.residues))
```

Comment:
This should be `pos - 1` to preserve directionality. The final call should be `omegaangle(res, res_prev)`. The same is true for `phiangle` (but not `psiangle`).